core/schema/directory.graphqls

```diff
@@ -64,10 +64,11 @@ type Directory {
     Path to the Dockerfile to use.
     Defaults to './Dockerfile'.
     """
-    dockerfile: String,
+    dockerfile: String
 
-    platform: Platform,
+    platform: Platform
     buildArgs: [BuildArg!]
+    target: String
```

Comment:
Not sure why these are missing doc strings, here's a backfill:

```suggestion
    "The platform to build."
    platform: Platform
    "Additional build arguments."
    buildArgs: [BuildArg!]
    "Target build stage to build."
    target: String
```

Reply:
@vito, rebased with your suggested changes + a rebuild of the SDKs